unison-src/transcripts/hello.output.md

```diff
@@ -83,10 +83,15 @@ hmm = "Not, in fact, a number"
 
 ```ucm
 
-  I found a value of type builtin.Text where I expected to find one of type builtin.Nat:
   
-      1 | hmm : .builtin.Nat
-      2 | hmm = "Not, in fact, a number"
+    ❓
+    
+    I couldn't resolve any of these symbols:
+    
+        1 | hmm : .builtin.Nat
+    
+    Using these fully qualified names: .builtin.Nat
+    
```

Comment:
@runarorama This is indicative of a problem, I think - the transcript calls `builtins.merge` at the start so it should be finding `builtin.Nat`. 

To debug, you can add a `.builtin> find` or `.builtin> ls` after the `builtins.merge`, just to see what the namespace looks like.

Reply:
This looks like an issue with name lookup in general of FQNs with absolute paths. E.g. if `.base.Nat` exists, it doesn't find it, but it does find `base.Nat`.